src/helpers/utils/markdown.tsx

```
@@ -0,0 +1,509 @@
+
+        <div className="relative">
+          <div
+            className={cn(
+              "font-mono text-sm leading-relaxed p-2",
+              isWrapped && "whitespace-pre-wrap break-words",
+              !isWrapped && "whitespace-pre overflow-x-auto"
+            )}
+            style={{
+              fontFamily: geistMono.style.fontFamily,
+            }}
+            dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)